daos/theaters.js

```diff
@@ -9,42 +9,42 @@ module.exports.getAllTheaters = async () => {
 
 
 module.exports.getMoviesByTheaterId = async (theaterId) => {
-    try {
-        const moviesInfo = await Theater.aggregate([
-            { $match: { _id: mongoose.Types.ObjectId(theaterId) }},
-            { $unwind: '$movies'},
-            { $lookup: { 
-                from: 'movies',
-                localField: 'movies',
-                foreignField: '_id',
-                as: 'movieInfo'
-            }},
-            { $unwind: '$movieInfo' },
-            { $group: { 
-                _id: '$movieInfo._id', 
-                title: { $first: '$movieInfo.title' },
-                actors: { $first: '$movieInfo.actors' }, 
-                genre: { $first: '$movieInfo.genre' },
-                synopsis: { $first: '$movieInfo.synopsis' },
-                rating: { $first: '$movieInfo.rating' },
-                releaseYear: { $first: '$movieInfo.releaseYear' },
-                moviePicUrl: { $first: '$movieInfo.moviePicUrl' }
-            }}
-        ]);
-        return moviesInfo;
-        }
-    catch (e) {
-        throw e;
+    if (!mongoose.Types.ObjectId.isValid(theaterId)) {
+        return null;
```

Comment:
This seems fine for our project, but I wonder if in a real site we'd throw an error here and let the route return a 400 for an invalid request?

Reply:
Ha, you spotted the shortcut right away. Yeah, here I was just trying to treat all requests with non-working IDs (both non-existing or simply invalid) the same way and send a 404. Just thought that would be simpler (Oh well I was just being lazy) :)